src/content/docs/docs/dotprompt.md

```diff
@@ -577,6 +577,8 @@ food items into all of your conversations.
 {{userQuestion}}
 ```
 
+Note that your final prompt must contain at least one `user` role.
```

Comment:
Do you think this should go into a callout?

https://starlight.astro.build/guides/authoring-content/#note-aside